forge-gui/res/cardsfolder/upcoming/vanish_into_eternity.txt

```diff
@@ -0,0 +1,6 @@
+Name:Vanish into Eternity
+ManaCost:2 W
+Types:Instant
+S:Mode$ RaiseCost | ValidCard$ Card.Self | Type$ Spell | Amount$ 3 | ValidTarget$ Creature | EffectZone$ All | Description$ This spell costs {3} more to cast if it targets a creature.
+A:SP$ ChangeZone | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Permanent.nonLand | TgtPromt$ Select target nonland permanant | StackDescription$ SpellDescription | SpellDescription$ Exile target nonland permanent.
```

Comment:
TgtPromt typo

Reply:
It's still there

Reply:
lol, i didn't realize i had two typos there. fixed.